app/static/src/app/userMessages.ts

```diff
@@ -66,6 +66,9 @@ export default {
             needsParamsToVary: "select at least one parameter to vary (Options tab)"
         }
     },
+    stochastic: {
+        individualTracesHidden: "Individual traces will be hidden"
```

Comment:
Do we have a pattern anywhere for having these strings be templates? I totally agree but wasn't sure how best to do this...

Reply:
Probably best just to make them functions rather than strings in the object in that case e.g. 
```
..
stochastic: {
   individualTracesHidden: (maxToShow: number) => `Individual traces will be hidden for models with more than ${maxToShow} replicates`
}
```